apps/cowswap-frontend/src/common/pure/ProtocolIcons/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { ProductVariant, UI, ProductLogo } from '@cowprotocol/ui'
+
+import { ProtocolIcon, ProtocolIconsContainer } from './styled'
+
+export interface SecondProtocolConfig {
+  icon: string
+  title: string
+  width?: number
+  height?: number
+}
+
+export interface ProtocolIconsProps {
+  secondProtocol: SecondProtocolConfig
+  showOnlyFirst?: boolean
+  showOnlySecond?: boolean
+  size?: number
+}
+
+const DEFAULT_ICON_SIZE = 18
+
+export function ProtocolIcons({ secondProtocol, showOnlyFirst, showOnlySecond, size }: ProtocolIconsProps) {
+  // If showing only first or only second, use the specified size or default to 36px for single icons
+  const iconSize = showOnlyFirst || showOnlySecond ? size || 36 : size
+
+  const logoHeight = iconSize ? iconSize * 0.75 : DEFAULT_ICON_SIZE
```

Comment:
`0.75` also needs to be moved in const

Reply:
Addressed in https://github.com/cowprotocol/cowswap/pull/5639